docs/geedocs/5.2.4/answer/3481558.html

```diff
@@ -69,26 +69,70 @@ <h2>Command reference</h2>
       </tr>
     </tbody>
   </table>
+
+
   <h2>geaddtoimageryproject</h2>
   <pre>
-<code><b>geaddtoimageryproject</b> [--<strong>mercator</strong> | --<strong>flat]</strong> [--<strong>historical_imagery</strong> |
---<strong>no_historical_imagery</strong>] -o <em>projectname</em>{[--<strong>maxlevel</strong> <em>level</em>] <em>insetresource</em>}</code>...</pre>
+<code><b>geaddtoimageryproject</b> [--<strong>mercator</strong> | --<strong>flat]</strong> [--<strong>historical_imagery</strong> | --<strong>no_historical_imagery</strong>]
+ -o <em>projectname</em> {[--<strong>maxleveloverride</strong> <em>level</em>]}
+ [--<strong>no_terrain_overlay</strong> | {--<strong>terrain_overlay</strong> --<strong>start_level</strong> <em>level</em> --<strong>resource_min_level</strong> <em>level</em> }]
+ <em>insetresource</em>} </code>...</pre>
+
+
   <h3>Purpose</h3>
   <p>Creates a new imagery project. This tool is capable of building Mercator imagery projects for 2D databases, or Flat (Plate Carrée) imagery projects with or without Historic Imagery Support for 3D databases.
```

Comment:
This looks like a pre-existing wording error, but we should go ahead and fix it. `geaddtoimageryproject` doesn't create a new project. It adds additional resources to an existing project. The wording should be updated for ge***project commands to reflect their functions.